pkg/kubelet/eviction/helpers.go

```diff
@@ -38,7 +38,7 @@ import (
 const (
 	unsupportedEvictionSignal = "unsupported eviction signal %v"
 	// the reason reported back in status.
```

Comment:
pkg/kubelet/eviction/helpers.go:40:2: comment on exported const Reason should be of the form "Reason ..."

Please fix this golint, the other changes looks good

Reply:
done

Reply:
Sorry for late response, fix lgtm